website/docs/cloud-docs/users-teams-organizations/teams.mdx

```diff
@@ -30,6 +30,7 @@ In addition to the HCP Terraform UI, you can use the following methods to manage
 - [Teams API](/terraform/cloud-docs/api-docs/teams) to list, create, update, and delete teams
 - [Team Members API](/terraform/cloud-docs/api-docs/team-members) to add and delete users from teams
 - [Team Tokens API](/terraform/cloud-docs/api-docs/team-tokens) to generate and delete tokens
+- [Organization Team Tokens API](/terraform/cloud-docs/api-docs/organization-team-tokens) to list tokens
```

Comment:
I don't think we need to mention this endpoint here because it doesn't actively _manage_ a team (though it'd definitely help you understand it better! but that's not what we are listing here)
```suggestion
```